CMakeLists.txt

```diff
@@ -237,20 +237,7 @@ if (INSTALL_PKGCONFIG_MODULE)
 	endif ()
 	if (ENABLE_MPEG)
 		set (EXTERNAL_MPEG_REQUIRE "libmpg123")
-		get_filename_component(MP3LAME_WE ${MP3LAME_LIBRARY} NAME_WE)
-		if (MP3LAME_HIP_LIBRARY)
-			get_filename_component(MP3LAME_HIP_WE ${MP3LAME_HIP_LIBRARY} NAME_WE)
-		endif ()
-		if (CMAKE_IMPORT_LIBRARY_PREFIX)
-			string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" MP3LAME_WE_NO_PREFIX ${MP3LAME_WE})
-			if (MP3LAME_HIP_LIBRARY)
-				string (REGEX REPLACE "^${CMAKE_IMPORT_LIBRARY_PREFIX}" "" MP3LAME_HIP_WE_NO_PREFIX ${MP3LAME_HIP_WE})
-			endif ()
-		endif ()
-		set (EXTERNAL_MPEG_LIBS "-l${MP3LAME_WE_NO_PREFIX}")
-		if (MP3LAME_HIP_LIBRARY)
-			set (EXTERNAL_MPEG_LIBS "${MPEG_LIBS} -l${MP3LAME_HIP_WE}")
-		endif ()
+		set (EXTERNAL_MPEG_LIBS "-lmp3lame")
```

Comment:
on Linux there's a pkg-config module (`lame.pc`), any reason you don't want to use that?

Reply:
The last time I looked they didn't have an official `lame.pc` file. The Vcpkg port does not generate it either. So I decided not to rely on it.

Reply:
ok